core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1959,13 +1967,19 @@ write_files:
                 securityContext:
                   privileged: true
                 volumeMounts:
+                - mountPath: /lib/modules
+                  name: lib-modules
+                  readOnly: true
                 - mountPath: /etc/kubernetes/kubeconfig
                   name: kubeconfig
                   readOnly: true
                 - mountPath: /etc/kubernetes/kube-proxy
                   name: kube-proxy-config
                   readOnly: true
               volumes:
+              - name: lib-modules
+                hostPath:
+                  path: /lib/modules
```

Comment:
Can we enclose this volume and the corresponding volumemount inside `{{if .Experimental.IPVSProxy.Enabled}}` if it is necessary when and only when IPVSProxy is enabled?